scripts/ci/check_python_signatures.py

```diff
@@ -0,0 +1,177 @@
+"""
+Compares the signatures in `rerun_bindings.pyi` with the actual runtime signatures in `rerun_bindings.so`.
+
+This does not check that the type annotations match. However, it does ensure that the number of arguments,
+the argument names, and whether the arguments are required or have defaults match between the stub and runtime.
+"""
+
+from __future__ import annotations
+
+import ast
+import importlib
+import inspect
+import sys
+from inspect import Parameter, Signature
+from pathlib import Path
+from typing import Any
+
+import parso
```

Comment:
Shouldn't this be added to the `pixi.toml`?

Reply:
It was already in the environment, but good call. I'll make it explicit.